### PR TITLE
Contrib - Configurable text expand repeater list styles

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
(cherry picked from contribution commit c94f820b45865968db0b5c0d6b685cc87ffcb13f)

Thanks @Blackbaud-SpencerMurphy!

### CODE REVIEW

**Review by Blackbaud-TrevorBurch — COMMENTED** [review]
@Blackbaud-SpencerMurphy - just a few minor things.

File: projects/layout/src/modules/text-expand-repeater/sky-text-expand-repeater-bullet-style-type.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export type SkyTextExpandRepeaterBulletStyleType = 'unordered' | 'ordered' | 'unstyled';
```
[the remark this answers is not in the document]
Comment (Blackbaud-SpencerMurphy) [reply projects/layout/src/modules/text-expand-repeater/sky-text-expand-repeater-bullet-style-type.ts]: Went ahead and updated the css classes as well

[the remark this answers is not in the document]
Comment (Blackbaud-SpencerMurphy) [reply projects/layout/src/modules/text-expand-repeater/sky-text-expand-repeater-bullet-style-type.ts]: I thought I needed to do something like this. I was trying to add it to the module which wasn't working and forgot to loop back around to it

File: projects/layout/src/modules/text-expand-repeater/text-expand-repeater.component.ts [hunk 2]
```diff
@@ -59,6 +63,13 @@ export class SkyTextExpandRepeaterComponent implements AfterViewInit, OnChanges
   @Input()
   public maxItems: number;
 
+  /**
+   * Specifies the style of bullet to use
+   * @default 'unordered'
+   */
+  @Input()
+  public listStyle: SkyTextExpandRepeaterBulletStyleType = 'unordered';
```
[the remark this answers is not in the document]
Comment (Blackbaud-SpencerMurphy) [reply projects/layout/src/modules/text-expand-repeater/text-expand-repeater.component.ts]: done

File: projects/layout/src/modules/text-expand-repeater/text-expand-repeater.component.scss [hunk 3]
```diff
@@ -8,3 +8,11 @@
   white-space: nowrap;
   margin-left: 40px;
 }
+
+.sky-text-expand-repeater-bullet-style-none {
```
[the remark this answers is not in the document]
Comment (Blackbaud-SpencerMurphy) [reply projects/layout/src/modules/text-expand-repeater/text-expand-repeater.component.scss]: done

**Comment by Blackbaud-TrevorBurch** [comment]
LGTM

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**